examples/hypervisor_cluster_profile.tf

```diff
@@ -0,0 +1,97 @@
+provider "oneview" {
+	ov_username = "<username>" 
+	ov_password = "<password>"
+	ov_endpoint = "<endpoint>"
+	ov_sslverify = false
+	ov_apiversion = <ov-apiversion>
+	ov_ifmatch = "*"
+}
+
+data "oneview_scope" "scope_obj" {
```

Comment:
scope_obj not being used in the resource creation below

Reply:
removed